wazo_agentd_client/exceptions.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# Copyright 2022 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from requests import HTTPError
+
+
+class AgentdError(HTTPError):
+    def __init__(self, response):
+        try:
+            body = response.json()
+        except ValueError:
+            raise InvalidAgentdError()
+
+        self.status_code = response.status_code
+        try:
```

Comment:
I'm not sure wazo-agentd errors fit this code. Here's the body I'm getting when testing a random API call in agentd.
```javascript
{
  "error": "no such queue"
}
```

Reply:
I also tested with `/status` and the only error I could get is a `401` which also uses the the `{"error": <reason>}` format

Reply:
yes you are right, I also tried it when I added the `no-auth` test scenario. I am not sure though what to do here...

Reply:
It think it would be easier to reuse the `_ResponseProcessor` from the agents.py file.

You could drop the AgentdCommand and use the RestCommand and then use response_processor.generic